src/app/content/highlights/reducer.ts

```diff
@@ -83,6 +85,9 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
       if (oldHiglightIndex < 0) { return state; }
 
       const oldHighlight = state.highlights[oldHiglightIndex];
+      const onlyColorChanged =
```

Comment:
this variable name would make sense if you weren't tacking on the current value of `hasUnsavedHighlight`, but as written maybe you should just call it `hasUnsavedHighlight`